frontend/src/main/angular/src/app/core/pages/games/games-with-files-card/games-with-file-copies-card.component.ts

```
@@ -81,29 +86,33 @@ export class GamesWithFileCopiesCardComponent implements OnInit, OnDestroy {
+      potentialFileCopyWithContext.progress = undefined;
+      console.debug("Updated potential file copy", event, potentialFileCopyWithContext);
     } else {
       console.warn("Could not find potential file copy", event, this.potentialFileCopiesWithContextByGameTitle);
     }
   }
 
-  private findPotentialFileCopy(event: FileCopyStatusChangedEvent): PotentialFileCopy | undefined {
+  private findPotentialFileCopyWithContext(fileCopyNaturalId: FileCopyNaturalId):
+    PotentialFileCopyWithContext | undefined {
     return Array.from(this.potentialFileCopiesWithContextByGameTitle?.values() ?? [])
       .flatMap(potentialFileCopiesWithContext => potentialFileCopiesWithContext)
```

Biome (1.9.4)

[error] Avoid unnecessary callback in flatMap call.

You can just use flat to flatten the array.
Safe fix: Replace unnecessary flatMap call to flat instead.


(lint/correctness/noFlatMapIdentity)